force-app/main/default/objects/Position__c/fields/Name__c.field-meta.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<CustomField xmlns="http://soap.sforce.com/2006/04/metadata">
+    <fullName>Name__c</fullName>
```

Comment:
why you created custom name field when there is standard field present while creating objects